### ISSUE TITLE
Accessibility test timing out on the pipeline

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#2475]
Pipeline runs are failing because the accessibility test is timing out., specifically when run on the chrome browser.
